TrafficCapture/captureOffloader/src/main/java/org/opensearch/migrations/trafficcapture/StreamChannelConnectionCaptureSerializer.java

```diff
@@ -56,11 +57,13 @@
  *   }
  * }
  * 3: 1
+ * }
+ * </pre>
  */
 @Slf4j
 public class StreamChannelConnectionCaptureSerializer<T> implements IChannelConnectionCaptureSerializer<T> {
 
-    private static final int MAX_ID_SIZE = 96;
+    private static final int MAX_ID_SIZE = 100;
```

Comment:
@gregschohn, let me know if we should increase this as 100 is exactly what we expect out of our realIds

Reply:
I don't have a strong opinion.  If I were doing this code from scratch, I'd probably put an assertion that I could create a new CodedOutputStream from the factory that was able to write the boilerplate stuff - all within a lambda inside the assert statement.